distribution/docs/src/main/resources/content/_tables/Session.adoc

```diff
@@ -19,7 +19,10 @@
 |Session Timeout (in minutes)
 |expirationTime
 |Integer
-|The number of minutes after a session has been inactive that it should be invalidated.
+|Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 31 minutes.
+
+See also: <<_ddf.platform.ui.config,Platform UI Config>>.
```

Comment:
👍 Nice update here. I think that should help prevent some confusion.
✏️ `See <<_ddf.platform.ui.config,Platform UI Config>>.`